src/DssSpell.sol

```diff
@@ -111,156 +133,7 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-        vat.file("Line", _sub(vat.Line(), lineReduction));
-
-
-        // Recognized Delegate Compensation
-        //    https://forum.makerdao.com/t/recognized-delegate-compensation-breakdown-may-2022/15536
-        //
-        //    Flip Flop Flap Delegate LLC - 12000 DAI - 0x688d508f3a6B0a377e266405A1583B3316f9A2B3
-        DssExecLib.sendPaymentFromSurplusBuffer(FLIPFLOPFLAP_WALLET, 12_000);
-        //    schuppi - 12000 DAI - 0xCCffDBc38B1463847509dCD95e0D9AAf54D1c167
-        DssExecLib.sendPaymentFromSurplusBuffer(SCHUPPI_WALLET, 12_000);
-        //    Feedblack Loops LLC - 12000 DAI - 0x80882f2A36d49fC46C3c654F7f9cB9a2Bf0423e1
-        DssExecLib.sendPaymentFromSurplusBuffer(FEEDBLACKLOOPS_WALLET, 12_000);
-        //    MakerMan - 11025 DAI - 0x9AC6A6B24bCd789Fa59A175c0514f33255e1e6D0
-        DssExecLib.sendPaymentFromSurplusBuffer(MAKERMAN_WALLET, 11025);
-        //    ACREInvest - 9372 DAI - 0x5b9C98e8A3D9Db6cd4B4B4C1F92D0A551D06F00D
-        DssExecLib.sendPaymentFromSurplusBuffer(ACREINVEST_WALLET, 9372);
-        //    monetsupply - 6275 DAI - 0x4Bd73eeE3d0568Bb7C52DFCad7AD5d47Fff5E2CF
-        DssExecLib.sendPaymentFromSurplusBuffer(MONETSUPPLY_WALLET, 6275);
-        //    JustinCase - 7626 DAI - 0xE070c2dCfcf6C6409202A8a210f71D51dbAe9473
-        DssExecLib.sendPaymentFromSurplusBuffer(JUSTINCASE_WALLET, 7626);
-        //    GFX Labs - 6607 DAI - 0xa6e8772af29b29B9202a073f8E36f447689BEef6
-        DssExecLib.sendPaymentFromSurplusBuffer(GFXLABS_WALLET, 6607);
-        //    Doo - 622 DAI - 0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0
-        DssExecLib.sendPaymentFromSurplusBuffer(DOO_WALLET, 622);
-        //    Flipside Crypto - 270 DAI - 0x62a43123FE71f9764f26554b3F5017627996816a
-        DssExecLib.sendPaymentFromSurplusBuffer(FLIPSIDECRYPTO_WALLET, 270);
-        //    Penn Blockchain - 265 DAI - 0x070341aA5Ed571f0FB2c4a5641409B1A46b4961b
-        DssExecLib.sendPaymentFromSurplusBuffer(PENNBLOCKCHAIN_WALLET, 265);
-
-
-        // Starknet Bridge Changes
-        // https://forum.makerdao.com/t/details-about-spells-to-be-included-in-june-8th-2022-executive-vote/15532
-        //
-        //    Increase Starknet Bridge Limit from 100,000 DAI to 200,000 DAI
-        StarknetLike(STARKNET_DAI_BRIDGE).setCeiling(200_000 * WAD);
-        //    Give DSChief control over L1EscrowMom
-        DssExecLib.setAuthority(STARKNET_ESCROW_MOM, DssExecLib.getChangelogAddress("MCD_ADM"));
-
-
-        // Changelog
-        DssExecLib.setChangelogAddress("STARKNET_ESCROW_MOM", STARKNET_ESCROW_MOM);
-        DssExecLib.setChangelogAddress("STARKNET_ESCROW", STARKNET_ESCROW);
-        DssExecLib.setChangelogAddress("STARKNET_DAI_BRIDGE", STARKNET_DAI_BRIDGE);
-        DssExecLib.setChangelogAddress("STARKNET_GOV_RELAY", STARKNET_GOV_RELAY);
-        DssExecLib.setChangelogVersion("1.13.1");
+        // DEBATE: we could just add the MKR distributions for next week here.
```

Comment:
The next spell only has two MKR distributions, we could just add them here.  We already decided to make them two separate spells, but figured I would ask one last time.

I'm leaning towards two spells still.

Reply:
There's precedent that an out-of-cycle executive should not include anything other than the specific urgent or emergent tasks.

Reply:
I also prefer two spells because there could hypothetically be debate about the distributions and this is urgent